src/services/orders.js

```diff
@@ -0,0 +1,149 @@
+const { Orders } = require('../models');
+const { OrderStatuses } = require('../models');
+const { Product } = require('../models');
+
+class OrdersService {
+  async createOrder (req, res) {
+    const orderStatus = await OrderStatuses.findOne({
+      name: 'In progress'
+    }).exec();
+    const order = new Orders(req.body);
+    order.status = orderStatus._id;
+    await order.save();
+    return res.json(true);
+  }
+
+  async getOrders (req, res) {
+    const { _id, status } = req.query;
+    const query = {};
+
+    if (_id) query._id = _id;
+    if (status) query.status = status;
```

Comment:
Instead of these all ifs you can do next:
Object.keys(req.query).forEach(key => {
  if (!req.query[key]) {
    delete req.query[key];
  }
});